packages/openneuro-app/src/scripts/datalad/fragments/accession-number.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from '@emotion/styled'
+import ToolTip from './copyable-tooltip.jsx'
+
+const Container = styled.div({
+  textAlign: 'right',
+  padding: '5px 10px',
+  color: '#565656',
+  fontSize: '12px',
+})
+
+const AccessionNumber = ({ datasetId }) => (
+  <div className="dataset-git-hash">
```

Comment:
Since the git hash and accession number components do the same thing, I think it'd be better to generalize one of them so that it can take whatever text and tip is necessary as props (and maybe the padding style too). Otherwise, we should at least change this classname.